provision_tools.sh

```diff
@@ -34,17 +34,15 @@ if command -v pyenv 1>/dev/null 2>&1; then
   eval "$(pyenv init -)"
 fi
 
-## Install the versions of Python that we use.
-export PY2_VER="2.7.16"
+## Install the version of Python that we use.
 export PY3_VER="3.6.9"
-echo -e "\033[44;97m!¡!¡! CFPB Mac Setup !¡!¡! INSTALLING PYTHON ${PY3_VER} AND ${PY2_VER} ¡!¡!¡\033[0m"
+echo -e "\033[44;97m!¡!¡! CFPB Mac Setup !¡!¡! INSTALLING PYTHON ${PY3_VER} ¡!¡!¡\033[0m"
 pyenv install ${PY3_VER}
-pyenv install ${PY2_VER}
 
 ## Set the global Python versions.
-echo -e "\033[44;97m!¡!¡! CFPB Mac Setup !¡!¡! SETTING GLOBAL PYTHON VERSIONS ¡!¡!¡\033[0m"
-echo "!¡!¡! (This will make python version ${PY3_VER}, and python2 will be ${PY2_VER}.)"
-pyenv global ${PY3_VER} ${PY2_VER}
+echo -e "\033[44;97m!¡!¡! CFPB Mac Setup !¡!¡! SETTING GLOBAL PYTHON VERSION ¡!¡!¡\033[0m"
+echo "!¡!¡! (This will make python version ${PY3_VER}.)"
```

Comment:
This line seems unnecessary and potentially confusing now that we're only installing one version.

```suggestion
```